dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThread.java

```diff
@@ -1512,6 +1515,10 @@ private void submitStandByTask() {
                     logger.info("remove task {},id:{} , because depend result : {}", task.getName(), task.getId(), dependResult);
                 }
             }
+            for (TaskInstance task : skipSubmitInstances) {
+                readyToSubmitTaskQueue.put(task);
```

Comment:
Why need to add back these skip task instances ?

Reply:
Please see #12291